tests/protocol/LiquidationTestToken.test.js

```diff
@@ -52,6 +53,9 @@ contract("ProtocolLiquidationTestToken", (accounts) => {
     let sovryn, SUSD, WRBTC, RBTC, BZRX, loanToken, loanTokenWRBTC, priceFeeds, SOV;
 
     async function deploymentAndInitFixture(_wallets, _provider) {
+        // need to deploy the mutex in the initialization. Otherwise, the global reentrancy prevetion will not be working & throw an error
```

Comment:
```suggestion  
        // Need to deploy the mutex in the initialization. Otherwise, the global reentrancy prevention will not be working & throw an error.
```